xmeffects.js

```diff
@@ -1,8 +1,14 @@
+(function (window) {
+if (!window.XMPlayer) {
+  window.XMPlayer = {};
+}
+var player = window.XMPlayer;
```

Comment:
Oh, I see. Either xmeffects.js or xm.js will create the `window.XMPlayer` and each will use the other's if it already exists. OK.